components/camel-aws/src/main/java/org/apache/camel/component/aws/sqs/SqsConfiguration.java

```diff
@@ -345,6 +347,17 @@ public int getConcurrentConsumers() {
     public void setConcurrentConsumers(int concurrentConsumers) {
         this.concurrentConsumers = concurrentConsumers;
     }
+	
+    /**
+     *  To define the queueUrl explicitly. All other parameters, which would influence the queueUrl, are ignored.
+     */
```

Comment:
Add better javadoc what the purpose of this is, eg for testing/mocking etc. That it should not be used normally.

Reply:
I tried to improve this. I hope the new text covers the purpose well enough.